### ISSUE TITLE
Add p-checker/p-curve compatible output option?

### ISSUE THREAD

**Issue by christinabergmann** [langcog/metalab-archive#111]
Got feedback on Twitter: https://twitter.com/nicebread303/status/786171215341719552 
Main suggestion regarded simple input (imo not feasible [right now]), but it might be useful to simplify interfacing with p-checker/p-curve. 

[p-curve](http://www.p-curve.com/app4/) example:
t(88)=2.1
r(147)=.246
F(1,100)=9.1
f(2,210)=4.45
Z=3.45
chi2(1)=9.1
r(77)=.47
chi2(2)=8.74

[p-checker](http://shinyapps.org/apps/p-checker/) example:
# Easy mode: only enter the test statistics with df

t(47) = 2.1
chi2(1) = 9.15
r(77) = .47
F(1, 88) = 9.21
p = .02
p(48) = .018
# add reported p-value; mark one-tailed; set alpha level

t(123) = 2.54; p < .01
Z = 1.9; one-tailed; p=.03
r(25) = 0.21; crit=.10
# add paper ID

A&B (2001) Study1: t(88)=2.1; one-tailed; p < .02
A&B (2001) Study1: r(147)=.246
A&B (2001) Study2: F(1,100)=9.1
CD&E (2014) S1a: F(1,210)=4.45; p < .01
CD&E (2014) S1b: t(123)=2.01; one-tailed; p = .02

**Comment by mcfrank** [comment]
Yes, shouldn't be too hard. Let's consider this as a possible enhancement.

Mike
